raiden/utils/typing.py

```diff
@@ -103,6 +103,9 @@ def typecheck(value: Any, expected: Type):
 T_FeeAmount = int
 FeeAmount = NewType("FeeAmount", T_FeeAmount)
 
+T_RelativeFeeAmount = int
+RelativeFeeAmount = NewType("RelativeFeeAmount", T_RelativeFeeAmount)
```

Comment:
This is a good place to document the reasonable range of values, and what they mean.

Maybe `1` means one-per-million?  Or not?  It's not easy to figure out.